src/main/java/com/thealgorithms/backtracking/KnightsTour.java

```diff
@@ -105,8 +98,7 @@ private static List<int[]> neighbors(int row, int column) {
         List<int[]> neighbour = new ArrayList<>();
 
         for (int[] m : moves) {
-            int x = m[0];
-            int y = m[1];
+            int x = m[0],y = m[1];
```

Comment:
Remove this change.

https://www.oracle.com/java/technologies/javase/codeconventions-declarations.html